specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext-authoring.json

```diff
@@ -1760,44 +1825,44 @@
           "type": "string"
         }
       },
-      "x-ms-client-name": "ExportedCustomMultiLabelClassificationDocument"
+      "x-ms-client-name": "ExportedCustomEntityRecognitionDocument"
     },
-    "TextAnalysisAuthoringExportedCustomMultiLabelClassificationProjectAssets": {
-      "description": "Represents the exported assets for a custom multi-label classification project.",
+    "TextAnalysisAuthoringExportedCustomEntityRecognitionProjectAssets": {
+      "description": "Represents the exported assets for a entity recognition project.",
       "type": "object",
       "allOf": [
         {
           "$ref": "#/definitions/TextAnalysisAuthoringExportedProjectAssets"
         }
       ],
       "properties": {
-        "classes": {
-          "description": "The list of classes in the project.",
+        "entities": {
+          "description": "The list of entities belonging to the project.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/TextAnalysisAuthoringExportedClass"
+            "$ref": "#/definitions/TextAnalysisAuthoringExportedEntity"
           }
         },
         "documents": {
-          "description": "The list of documents in the project.",
+          "description": "The list of documents belonging to the project.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/TextAnalysisAuthoringExportedCustomMultiLabelClassificationDocument"
+            "$ref": "#/definitions/TextAnalysisAuthoringExportedCustomEntityRecognitionDocument"
           }
         }
       },
-      "x-ms-discriminator-value": "customMultiLabelClassification",
-      "x-ms-client-name": "ExportedCustomMultiLabelClassificationProjectAssets"
+      "x-ms-discriminator-value": "CustomEntityRecognition",
+      "x-ms-client-name": "ExportedCustomEntityRecognitionProjectAssets"
     },
-    "TextAnalysisAuthoringExportedCustomNamedEntityRecognitionDocument": {
-      "description": "Represents an exported document for a custom named entity recognition project.",
+    "TextAnalysisAuthoringExportedCustomMultiLabelClassificationDocument": {
+      "description": "Represents an exported document of a custom multi-label classification project.",
       "type": "object",
       "properties": {
-        "entities": {
-          "description": "The list of entity labels belonging to the document.",
+        "classes": {
```

Comment:
This could pose a problem. Words like "Type" or "Class" are forbidden from the SDK. Their plural forms may be acceptable but seem quite iffy. "entity" or "entities" is more common. Why the change?

Reply:
This is just a re-order of the schemas in the swagger file. The original schema is still there.